cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -659,14 +659,14 @@ instance (Condense (HeaderHash blk), ProtocolLedgerView blk, SupportedBlock blk,
     TraceFoundIntersection _ _ _ ->
       mkObject [ "kind" .= String "ChainSyncClientEvent.TraceFoundIntersection" ]
 
-instance StandardHash blk => ToObject (TraceChainSyncServerEvent blk b) where
+instance (Show b, StandardHash blk) => ToObject (TraceChainSyncServerEvent blk b) where
     toObject _ ev = case ev of
-      TraceChainSyncServerRead tip _ ->
+      TraceChainSyncServerRead _ update ->
         mkObject [ "kind" .= String "ChainSyncServerEvent.TraceChainSyncServerRead"
-                 , "block" .= (String (pack . show $ tipPoint tip)) ]
```

Comment:
You can still include the tip in the object. So restore this line, but use `"tip"` as the key.